src/platform-includes/configuration/integrations/pluggable-integrations/node.mdx

```diff
@@ -0,0 +1,76 @@
+Pluggable integrations are integrations that can be additionally enabled, to provide specific features. They're documented so you can see what they do and that they can be enabled. To enable pluggable integrations, provide a new instance with your config to the `integrations` option. For example: `integrations: [new Sentry.Integrations.Modules()]`.
+
+## Core
+
+### CaptureConsole
+
+_Import name: `Sentry.Integrations.CaptureConsole`_
+
+This integration captures all `Console API` calls and redirects them to Sentry using `captureMessage` call.
+It then retriggers to preserve default native behavior.
+
+Available options:
+
+```javascript
+{
+  levels: string[]; // an array of methods that should be captured, defaults to ['log', 'info', 'warn', 'error', 'debug', 'assert']
+}
+```
+
+### Dedupe
+
+_Import name: `Sentry.Integrations.Dedupe`_
+
+This integration deduplicates certain events. It can be helpful if you are receiving many duplicate errors. Be aware that we will only compare stack traces and fingerprints.
```

Comment:
```suggestion
This integration deduplicates certain events. It can be helpful if you're receiving many duplicate errors. Be aware that we will only compare stack traces and fingerprints.
```